server/liveblog/themes/template/filters.py

```diff
@@ -71,7 +70,20 @@ def addten(date):
 
 
 def ampify(html):
-    if re.search('iframe', html, re.IGNORECASE):
+    brightcove_re_array = [
+        'https?://(www\.)?players\.brightcove\.net/',
```

Comment:
the issue with the previous string was that it didn't started with proper url.`https?://(www\.)?`